MANIFEST.in

```diff
@@ -1,3 +1,5 @@
+include LICENSE.txt
```

Comment:
I don't think you need to manually include LICENSE.txt and requirements.txt because of `recursive-include ... *.txt`

Reply:
Thanks.

Reply:
I just found that recursive-include includes all *.txt files under qiskit_optimization. This line is necessary.

Reply:
sorry. I misunderstood the spec. You are correct. https://packaging.python.org/guides/using-manifest-in/#manifest-in-commands